python/whylogs/api/writer/writer.py

```diff
@@ -20,7 +22,7 @@ def get_default_path(self) -> str:
         pass
 
     @abstractmethod
-    def write(self, path: Optional[str] = None, **kwargs: Any) -> None:
+    def write(self, path: Optional[str] = None, **kwargs: Any) -> requests.Response:
```

Comment:
Two issues:
1. We can't reference an optional dependency's type ('requests' package is only used in whylogs[whylabs]) in our base abstract classes used across all Writers/Writables
2. I would expect the write() method would have the same return type across Writer and Writable

Reply:
1. Got it. 
2. That's fair, that makes sense. Hmm ok, well then. One option is that we return Any for all writables. That's a little odd but also kinda understandable. We could return an int for all writes which would be more understandable with the error codes. 

Or a wrapper around the write just for whylabs. I don't like this. 

Or lastly, not add the response which would force use of the private method _do_upload instead. Also do not like. 

My favorite would be the return any or int. Thoughts?

Reply:
`Any` works, or maybe something like `Tuple[bool, str]` if this is just to give success/fail status + some debug / status information. The string part could be: filepath, url, in success case, or error or http status code in the failure case?

Reply:
Didn't see this before latest commit. I had moved it to an Optional[int] where most don't return anything. I can iterate and move it to that. I thought the direct response of the status code would be nice, but I agree that has more information to give